### ISSUE TITLE
Autocomplete for PostgreSQL stopped working

### ISSUE THREAD

**Issue by nightmarebadger** [mtxr/SQLTools#61]
I have a feeling this is not an actual issue but something with my setup instead ... however it was working fine up until a little while ago and I can't seem to get it back so thought I would raise it. 

I am working with PostgreSQL using the extended syntax highlighting here: https://github.com/tkopets/sublime-postgresql-syntax

It used to work with autocompletion (table & column names etc.) perfectly but has stopped some time ago. If I change the syntax to base SQL it still works perfectly, but I miss some of the extended syntax highlighting (like `vacuum` and `analyze`). 

As far as I know I haven't changed anything in my setup. Any help in getting it to work again would be appreciated :).

**Comment by mtxr** [comment]
Hello.

Does it still happening? Check please if you have any stack trace in your console. 

Syntax highlighting currently is not provided by SQLTools. I found some useful plugins that do this job perfectly. I can't see SQLTools embracing this feature in a near future because existing plugins are really good.

**Comment by Alexey-T** [comment]
Maybe this is reason
https://github.com/mtxr/SQLTools/commit/2c5cdb46062b859faa8efaf936c71bbde0f31cb7#diff-524df139ebf1ccfeb7afb11238405a2e

**Comment by tkopets** [comment]
@nightmarebadger 
tl;dr
add the line below to your `SQLTools.sublime-settings` and you will see the completions:
```
    "selectors": ["source.sql", "source.pgsql"]
```

Actually, the completions are supplied only when the selector matches those listed in settings, see [SQLTools.py line 246](https://github.com/mtxr/SQLTools/blob/a99269b590d4de3c5138e9b01b2440d2adddb4e1/SQLTools.py#L246) and few lines below.

To make completions to show up when using custom syntax highlight e.g. (https://github.com/tkopets/sublime-postgresql-syntax) just list this custom syntax highlight selector in SQLTools settings.

In your case, the selector would be [source.pgsql](https://github.com/tkopets/sublime-postgresql-syntax/blob/936dfe8e1ef2f72242892b40262289b2416fe828/PostgreSQL.tmLanguage#L563), so if you add the line below to `SQLTools.sublime-settings` you will get the completions.
```
    "selectors": ["source.sql", "source.pgsql"]
```

@mtxr 
It looks like there are only a few additional SQL syntax highlighters available via Package Control, so I would propose to include all of them in default SQLTools config, so those people who use them get autocompletions working out of the box.
I will review those few additional SQL syntax highlights and their selectors and will prepare a PR to include them.

**Comment by nightmarebadger** [comment]
Thanks for this @tkopets! It looks like a new release was made in the mean time too as the Autocomplete works fine with my PostgreSQL syntax highlighting again so I didn't even have to change anything. Just wanted to confirm it worked fine :)
